app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.framgia.fsalon"
+          xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-permission android:name="android.permission.INTERNET"/>
+
+    <application
```

Comment:
WARNING: [GoogleAppIndexingWarning] App is not indexable by Google Search; consider adding at least one Activity with an ACTION-VIEW intent filter. See issue explanation for more details.
 Adds URLs to get your app into the Google index, to get installs and traffic to your app from Google Search.